cart-service/dockerfiles/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM node:12-alpine as build
+WORKDIR /app
+COPY ./package*.json ./
+RUN npm install
+COPY . ./
+RUN npm run build
+
+
+FROM node:12-alpine
+WORKDIR '/app'
+COPY --from=build /project/dist /project/dist
```

Comment:
At this line docker break work with message that **/project/dist** folder doesn't exist.

Reply:
Thanks, yes, sorry, I forgot to push the latest version, now it is fixed.

Reply:
Ok. now it works
![image](https://user-images.githubusercontent.com/9920913/101614933-964d2480-3a1e-11eb-9ef5-74960f621698.png)